### DISCUSSION TITLE
How to configure Wine for my game ?

### DISCUSSION THREAD

**Discussion by alizeegagnon** [legendary-gl/legendary#293] [Q&A]
Hello ! I just installed Legendary on my Arch Linux machine. I installed the game Overcooked 2. When launching the game I can't input anything with the keyboard and I'm stuck at the start screen. It's a problem I've encountered in other games through Steam and I'm trying to follow this Stack Overflow thread: [link](https://askubuntu.com/questions/547430/keyboard-not-working-with-games-in-wine)

How can I get more information about the wine version or settings Legendary is using ? The command "legendary --egl-wine-prefix" does not work for me (it just outputs the help message). Particularly I'd like to know which windows version Wine is emulating (7, 8, 8.1, 10... ?)

Thanks a lot to anybody reading this !

**Comment by CommandMC** [comment]
Legendary uses your system Wine if you haven't changed that setting yet, so you're most likely using 6.11. Overcooked 2 requires enabling virtual desktop in `winecfg`, see [my EpicLinux wiki](https://github.com/CommandMC/EpicLinux/wiki/Fixes-for-common-issues-and-Recommendations#3-try-the-virtual-desktop-feature) for more info

**Comment by xcom169** [comment]
Is it possible to use WINE GE instead of system wine?
